docs/zh/latest/config.json

```diff
@@ -116,9 +116,8 @@
               "type": "category",
               "label": "Metrics",
               "items": [
-                "plugins/promethues",
-                "plugins/node-status",
-                "plugins/datadog"
```

Comment:
Why remove it?

Reply:
![image](https://user-images.githubusercontent.com/3895272/154944184-67f691f3-3c9f-4a74-b8ed-5d66fcef1cdf.png)

Reply:
It seems that we should add the Chinese doc, I have submitted an issue to track this: https://github.com/apache/apisix/issues/6406.

Reply:
yes,Yes. But it shouldn't be done in this PR.

Reply:
master branch already fix .